src/Storage/Storage/Storage.csproj

```diff
@@ -12,8 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0-preview.6" />
-    <PackageReference Include="Azure.Storage.Files.DataLake" Version="12.3.0-preview.2" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0-dev.20200811.1" />
+    <PackageReference Include="Azure.Storage.Files.DataLake" Version="12.3.0-dev.20200811.1" />
```

Comment:
local packages

Reply:
This is private signed XSCL. I have added it to localfeed. So the CI will pass and PSH still works well.

As storage team and PSH team before agreement:
1. For preview module, we can release with private signed SDK. Like following PR:
https://github.com/Azure/azure-powershell/pull/10424
2. For stable PSH, we can temporarily release PowerShell cmdlets based on a private, signed package, with the next major release picking up the released nuget package